sdk/lusid/utilities/api_configuration.py

```diff
@@ -18,7 +15,7 @@ def __init__(self, token_url=None, api_url=None, username=None, password=None, c
         :param str certificate_filename: Name of the certificate file (.pem, .cer or .crt)
         :param lusid.utilities.ProxyConfig proxy_config: The proxy configuration to use
         """
-        self.token_url = token_url
+        self.__token_url = token_url
```

Comment:
This was `self.token_url` so that it uses the setter function in the function with the `@token_url.setter` decorator. With this change that is completely bypassed.

Reply:
None of the other variables have setter functions which is why they set the private variable directly using self.__api_url = api_url.